jekyll/_cci2/deployment-integrations.md

```diff
@@ -348,3 +348,49 @@ workflows:
     git push --follow-tags
     ``` 
 5.  If tests passed, CircleCI will publish the package to npm automatically.
+
+## SSH
+
+To configure CircleCI
+to deploy your application over SSH,
+follow the steps below.
+
+1. Add the SSH key for the server
+to which you're deploying.
```

Comment:
deploying. -> deploying on the Settings > SSH Permissions page of the CircleCI app.

Reply:
So, rather than explicitly name where the action is happening, I think it makes more sense to send people to a single source of truth. This cuts down on duplication and saves us from having to update instructions in several areas.

Reply:
We don't make choices like this based on our maintenance needs though. We make choices that are best for the reader. In this case, it is best to mention that this is done in the app and give them the page name without making them go to three different articles to find it out.